changelogs/fragments/2779_redhat_subscription-add_server_prefix_and_server_port.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - redhat_subscription - add ``server_prefix`` and ``server_port`` paramaters (https://github.com/ansible-collections/community.general/pull/2779).
```

Sorry, I never learn the spell of this damned word!
```suggestion
  - redhat_subscription - add ``server_prefix`` and ``server_port`` parameters (https://github.com/ansible-collections/community.general/pull/2779).
```

---

Ooops, I completely missed that...

---

These two parameters would not be used frequently so I did not plan to add examples on them. Thank you. (I also missed to spot the 'paramaters' there)